css/components/loading-dialog.css

```diff
@@ -0,0 +1,73 @@
+.loading-dialog {
+  --fade-out-duration: 0.4s;
+  /* take up the full screen */
+  position: absolute;
+  inset: 0;
+
+  /* reset browser defaults for <dialog> elements */
+  border: none;
+  width: 100%;
+  height: 100%;
+  max-width: unset;
+  max-height: unset;
+  padding: 0;
+
+  /* cover other interface elements like buttons */
+  z-index: 1;
+
+  background: var(--theme-bg, var(--color-blue-70));
+  color: var(--color-white);
+
+  /* progressive enhancement: in supported browsers, fade out loading screen when the start button is pressed */
+  transition: opacity var(--fade-out-duration), display var(--fade-out-duration) allow-discrete;
+
+  &[open] {
+    opacity: 1;
+  }
+
+  &:not([open]) {
+    opacity: 0;
+  }
+}
+
+.loading-dialog__inner {
+  width: 80%;
+  height: 100%;
+  max-width: 500px;
+  margin-inline: auto;
+
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 2rem;
+
+  :where(img) {
+    /* TODO: determine if/how to show app icon */
+    display: none;
+    max-width: 200px;
+  }
```

Comment:
with the changes we've made, drawing apps load too fast to make the 'Loading...' and the icon readable, it's just an awkward flash. I've therefore hidden the image for now, you can decide if you want to display it statically about the button or whatever you like.

It is still useful having the loading dialog because loading is _fast_, but it's still not _instant_, so the dialog covers any weird flashes that might be seen. We also use the 'click to start' button as the user interaction that grants us permission to play audio (the web platform doesn't like autoplaying audio, with good reason)